tailwind.config.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
         "blue-gradient":
           "linear-gradient(294.77deg, #0A67C7 50.32%, rgba(0, 114, 231, 0.67) 100%)",
       },
+      fontFamily: {
+        canada : ['Radio Canada', 'sans-serif']
```

Comment:
É uma boa prática adicionar props como `fontFamily` no Tema (principal), pois assim ocorre a sobreescrita no tema como um todo.
Acima do `extend` ali, só mover esse bloco (objeto) com as alterações que fiz pra lá. Se tiver dúvidas, só falar aqui.
```suggestion
        sans: ["Inter", "sans-serif"],
        canada : ['Radio Canada', 'sans-serif'],
```